src/erl_cache_server.erl

```diff
@@ -27,17 +27,20 @@
 -export([init/1, handle_call/3, handle_cast/2, handle_info/2,
          terminate/2, code_change/3]).
 
--ifdef(namespace_types).
```

Comment:
As long as you're killing this, it looks like the sole reference here. can you remove the `platform_define` in the `rebar.config`?